characterize.py

```diff
@@ -139,9 +149,18 @@ def execute(self, request: ServiceRequest) -> None:
             }
         }
 
-        ResultSection(f"File entropy: {round(entropy, 3)}", parent=request.result, body_format=BODY_FORMAT.GRAPH_DATA,
+        entropy_sec = ResultSection(f"File entropy: {round(entropy, 3)}", parent=request.result, body_format=BODY_FORMAT.GRAPH_DATA,
                       body=json.dumps(entropy_graph_data))
 
+        if get_byte_histogram:
+            histogram_data = { 'entropy_byte_histogram' : histogram.tolist() }
+            histo_subsec = ResultSection(
```

Comment:
if you're setting the `ResultSection` parent to entropy_sec, then you don't need to set the variable `histo_subsec` since it is never used.

Reply:
options include `_ = ResultSection(...)` or just `ResultSection(...)`